File: api/src/main/java/api/security/config/JwtSettings.java
package api.security.config;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

@Component
public class JwtSettings {

    @Value("${jwt.security.secret}")
    private String jwtSecret;
    @Value("${jwt.access.expirationInMs}")
    private long jwtExpirationInMs;
    @Value("${jwt.refresh.expirationInMs}")
    private long jwtRefreshExpirationInMs;

    public String getJwtSecret() {
        return jwtSecret;
    }

    public long getJwtExpirationInMs() {
        return jwtExpirationInMs;
    }

    public long getJwtRefreshExpirationInMs() {
        return jwtRefreshExpirationInMs;
    }
}

File: api/src/main/java/api/security/config/JwtSettings.kt
package api.security.config

import org.springframework.beans.factory.annotation.Value
import org.springframework.stereotype.Component

@Component
class JwtSettings {

    @Value("\${jwt.security.secret}")
    val jwtSecret: String? = null
    @Value("\${jwt.access.expirationInMs}")
    val jwtExpirationInMs: Long = 0
    @Value("\${jwt.refresh.expirationInMs}")
    val jwtRefreshExpirationInMs: Long = 0
    @Value("\${jwt.registration.expirationInMs}")
    val jwtRegistrationExpirationInMs: Long = 0
}
